### DISCUSSION TITLE
Command to Check Country of Steam Account

### DISCUSSION THREAD

**Discussion by ksardas2015** [chr233/ASFEnhance#352] [功能建议 / Ideas]
It would be helpful to have a command that shows the current country of a Steam account. This would allow users to easily determine which accounts are eligible for certain purchases or gift-giving, saving time and avoiding potential issues with regional restrictions.

**Comment by chr233** [comment]
the only way to get account's region is from wallet symbol

**Reply by felipe19930** [reply]
Are you sure? How about https://store.steampowered.com/account/store_transactions/?


![image](https://github.com/user-attachments/assets/56137bea-f9ed-477b-a745-f23c044315b6)
